src/main/groovy/org/transmartproject/core/dataquery/highdim/tworegion/EventValues.groovy

```diff
@@ -0,0 +1,10 @@
+package org.transmartproject.core.dataquery.highdim.tworegion
+
+/**
+ * Represents metadata about one event, possibly linking to multiple junctions
+ */
+interface EventValues {
+    String getCgaType();
+    String getSoapClass();
```

Comment:
These come directly from CGA tools and SOAP Fusion so I don't think it should be an enum. If they add a type or a class upload would fail. They are documented in DDL SQL statements, I'll copy the doc here